### DISCUSSION TITLE
How can I configure Flameshot to take full screenshot and also save it to a location just pressing PrtSc key?

### DISCUSSION THREAD

**Discussion by formlessdao** [flameshot-org/flameshot#2492] [Q&A]
How can I configure Flameshot to take full screen screenshot and also save it to a location just pressing PrtSc key?

**Comment by mmahmoudian** [comment]
To teach you fishing:
1. Read https://flameshot.org/docs/advanced/commandline-options/
2. Create a shortcut for a command you want

To give you a fish:
1. Create a keyboard shortcut for `flameshot full --path /the/path/to/the/folder/`

P.s: I assumed you use Linux

**Reply by formlessdao** [reply]
I'm new to Linux so I appreciate you taking the time to teach me both, rarely do I stumble on such helpful people. Thanks!

**Reply by mmahmoudian** [reply]
This gets off-topic, but there are good resources to get help or search for answers.

If you have Flameshot questions you can ask on Github or on [our Matrix channels](https://matrix.to/#/%23flameshot-org:matrix.org). If your question is linux related, apart from [Ask Ubuntu](https://askubuntu.com/), [Server Fault](https://serverfault.com/), and [Super User](https://superuser.com/), you and always ask questions on various Matrix channels, or post your question on Mastodon (there are [instances with focus on Linux](https://mastodon.help/instances?lang=112&desc=Linux&minu=10&maxu=30000&minau=10&minki=500&black=1&creg=1&appr=0&lcok=0&ord=rand&advc=0&p=1))

Alternatively you can ask me questions via Matrix ([@mehrad:kde.org](https://matrix.to/#/@mehrad:kde.org)) or XMPP (mehrad@snopyta.org)

**Reply by formlessdao** [reply]
Hi I thanked you at that time but did not attempt to implement it then because I was busy. 
My actual question is how do you make that shortcut that executes the terminal commands you wrote?
I basically want to just press Prtscr key and that command is executed, screenshot of the full screen is made and saved to the location.
I want also no confirmation screen pop up like it does now asking me if I want to save.

I tried to make a custom shortcut command in Linux Mint but it does not work . I have an error : "Failed to execute child process 'flameshot' (no such file or directory)" in spite of me adding a valid location.
This is the command I used : "flameshot full --path /home/PcName/Pictures"

**Reply by mmahmoudian** [reply]
> I want also no confirmation screen pop up like it does now asking me if I want to save.

This indicates that you are a Gnome 41 Wayland user. There is no way around it for the time being. For any screenshot app in Gnome, they force this confirmation window to open. Either change your DE from Gnome to something else or take your complaints to gnome people.

**Reply by formlessdao** [reply]
> > I want also no confirmation screen pop up like it does now asking me if I want to save.
> 
> This indicates that you are a Gnome 41 Wayland user. There is no way around it for the time being. For any screenshot app in Gnome, they force this confirmation window to open. Either change your DE from Gnome to something else or take your complaints to gnome people.

I'm not, I use Cinnamon on Linux Mint.

**Reply by mmahmoudian** [reply]
Cinnamon should not ask for confirmation. Can you post a screenshot here of the confirmation dialogue you are seeing? also please provide some info about your OS: version, Cinnamon version, etc.

**Comment by marauderin** [comment]
Hello, I have the exact same issue, 

I am trying to bind Flameshot to Print screen button.

I tried several guides with the same problem:
https://www.addictivetips.com/ubuntu-linux-tips/flameshot-default-linux-screenshot-app/

"failed to execute child process Flameshot no such file or directory"
![image](https://user-images.githubusercontent.com/5735322/217795630-96e84714-d49d-4e8d-bd73-b7a85d552009.png)
![image](https://user-images.githubusercontent.com/5735322/217795778-0b05d27e-94d3-43d6-afd1-c8c8cb847a99.png)
![image](https://user-images.githubusercontent.com/5735322/217795887-cff39d50-0a47-414e-82d5-b16a6baaf567.png)

**Reply by mmahmoudian** [reply]
Make sure the binary you are trying to call (in this case `flameshot`) is in your `PATH`. One way to check that is to do `which flameshot`. 

![image](https://user-images.githubusercontent.com/390889/218025584-fab4fea3-8caf-45b9-8999-b5b56a668a0e.png)

If you see something different that the picture above, you either don't have Flameshot installed, or it is installed somewhere that is not in the `PATH` (try `echo ${PATH}` to see all places that are discoverable in your `PATH`). For example flatpaks are not in the `PATH` and you have to invoke the program via `flatpak run org.flameshot.Flameshot` as we have explained here:

https://flameshot.org/docs/installation/installation-linux/#flatpak

As a side note, I would also suggest you to always include the software version and the installation method when you are asking for help of some sort. There are many things that can go wrong or behave differently, and by not providing such specific information, there is much less incentive for the community to engage and try to help.

**Reply by marauderin** [reply]
Thank you very much, it solved my issue.

Also, my apologies for not including the software version. I will keep it in mind in the future.
